missilewars-plugin/src/main/java/de/butzlabben/missilewars/commands/MWCommands.java

```diff
@@ -51,77 +48,115 @@ public void mwCommand(CommandSender sender) {
+        sendHelpMessage(sender, "mw.vote", "/mw vote", "Vote for a arena.");
+        sendHelpMessage(sender, "mw.change", "/mw change <1|2>", "Changes your team.");
+        sendHelpMessage(sender, "mw.quit", "/mw quit", "Quit a game.");
+
+        sendHelpMessage(sender, "mw.stats", "/mw stats [from] [arena]", "Shows stats.");
+        sendHelpMessage(sender, "mw.stats.recommendations", "/mw stats recommendations [from] [arena]", "Shows recommendations.");
+        sendHelpMessage(sender, "mw.stats.players", "/mw stats players [from] [arena]", "Shows player list.");
+        sendHelpMessage(sender, "mw.stats.list", "/mw stats list [from] [arena]", "Lists history of games.");
+
+        sendHelpMessage(sender, "mw.listgames", "/mw listgames", "List the active games.");
+        sendHelpMessage(sender, "mw.paste", "/mw paste <missile>", "Pastes a missile.");
+        sendHelpMessage(sender, "mw.start", "/mw start [lobby]", "Starts the game.");
+        sendHelpMessage(sender, "mw.stop", "/mw stop [lobby]", "Stops the game.");
+        sendHelpMessage(sender, "mw.appendrestart", "/mw appendrestart [lobby]", "Appends a restart after the next game ends.");
+        sendHelpMessage(sender, "mw.reload", "/mw reload", "Reload the plugin.");
+        sendHelpMessage(sender, "mw.debug", "/mw debug", "Show debug info.");
+        sendHelpMessage(sender, "mw.restartall", "/mw restartall", "Restart all games.");
+
+        sendHelpMessage(sender, "mw.setup", "/mw setup <main|lobby|arena> <value> <set|teleport> [lobby]", "Setup the MW Locations or the lobby/arena locations.");
+    }
+    
+    @Subcommand("listgames|list|games")
+    @CommandCompletion("@nothing")
+    @CommandPermission("mw.listgames")
+    public void listgamesCommand(CommandSender sender, String[] args) {
+
+        sender.sendMessage(Messages.getPrefix() + "Current games:");
+
+        for (Game game : GameManager.getInstance().getGames().values()) {
+            sender.sendMessage("§e " + game.getLobby().getName() + "§7 -- Name: »" + game.getLobby().getDisplayName() + "§7« | Status: " + game.getState());
+            sender.sendMessage("§8 - §f" + "Load with startup: §7" + game.getLobby().isAutoLoad());
+            sender.sendMessage("§8 - §f" + "Current Arena: §7" + game.getArena().getName() + "§7 -- Name: »" + game.getArena().getDisplayName() + "§7«");
+            sender.sendMessage("§8 - §f" + "Total players: §7" + game.getPlayers().size() + "x");
+            sender.sendMessage("§8 - §f" + "Team 1: §7" + game.getTeam1().getColor() + game.getTeam1().getName()
+                    + " §7with " + game.getTeam1().getMembers().size() + " players");
+            sender.sendMessage("§8 - §f" + "Team 2: §7" + game.getTeam2().getColor() + game.getTeam2().getName()
+                    + " §7with " + game.getTeam2().getMembers().size() + " players");
+        }
+
     }
 
     @Subcommand("paste")
-    @Description("Pastes a missile.")
-    @Syntax("/mw paste <missile>")
```

Comment:
`@Description` and `@Syntax` aren't used, currently. It is a feature of the Command Framework. But we prefer to build the help command ourselves.